detekt.yml

```diff
@@ -36,7 +36,7 @@ complexity:
     threshold: 300
     active: false
   ComplexMethod:
```

Comment:
Does it make sense to remove this check? Not sure if we're getting any value from it. Looks like one of those you end up raising over and over when they blame.

Reply:
I raised the threshold of this check because the `FreeApplicative.foldMap` triggers it but it's necessary that it remains all in one function because it's inside a big mutable loop for performance reasons.